polyfill.d.ts

```diff
@@ -56,12 +56,38 @@ interface ResultConstructor {
   /**
    * Creates a result for a successful operation
    */
-  ok<V>(value: V): Result<V>
+  ok<V>(this: void, value: V): Result<V>
 
   /**
    * Creates a result for a failed operation
    */
-  error<V>(error: unknown): Result<V>
+  error<V>(this: void, error: unknown): Result<V>
+
+  /**
+   * Runs a function and wraps the result into a {@linkcode Result}.
+   *
+   * @example
+   *
+   * const [ok, error, value] = Result.try(func, arg1, arg2)
+   * const [ok, error, value] = await Result.try(asyncFunc, arg1, arg2)
+   * const [ok, error, value] = await Result.try(async (arg) => arg, 'pass')
```

Comment:
Javascript typically uses arg1, arg2, ... to notate variables that pass through, so examples 1 and 2 already seem to obviously pass through. This third example confuses the issue. It looks like a instruction, but I think you're just saying it's a variable that passes through.